static/app/utils/useProjectSdkNeedsUpdate.tsx

```diff
@@ -18,9 +17,20 @@ function useProjectSdkNeedsUpdate({
   | {isError: false; isFetching: true; needsUpdate: undefined}
   | {isError: true; isFetching: false; needsUpdate: undefined}
   | {isError: false; isFetching: false; needsUpdate: boolean} {
-  const {data, isLoading, isError} = useApiQuery<ProjectSdkUpdates[]>(
-    [`/organizations/${organization.slug}/sdk-updates/`],
-    {staleTime: 5000}
+  const path = `/organizations/${organization.slug}/sdk-updates/`;
+  const api = useApi({persistInFlight: true});
+  const {data, isLoading, isError} = useQuery(
+    [path],
+    async () => {
```

Comment:
If I were stumbling across this code I'd appreciate a comment about why this is necessary